perftest/src/perftest_data.rs

```diff
@@ -1,4 +1,4 @@
-// This file is generated by rust-protobuf 2.8.0. Do not edit
+// This file is generated by rust-protobuf 2.8.1. Do not edit
```

Comment:
Do we need this file checked in? I couldn't tell when I removed a bunch of other generate files if this one could go.

Reply:
no need to check this one I think

Reply:
Done.